pyopenms_viz/_core.py

```diff
@@ -1500,3 +1499,186 @@ def _get_plot_backend(backend: str | None = None):
     module = _load_backend(backend_str)
     _backends[backend_str] = module
     return module
+
+
+def plot_chromatogram(data: pd.DataFrame, x: str, y: str, backend: Optional[str] = None, **kwargs):
+    """
+    Plot a chromatogram using a seaborn-like API.
+
+    Parameters
+    ----------
+    data : pandas.DataFrame
+        The data to plot.
+    x : str
+        The column name for the x-axis data.
+    y : str
+        The column name for the y-axis data.
+    backend : str, optional
+        The backend to use for plotting. If not provided, it will try to use
+        matplotlib, bokeh, or plotly in that order.
+    **kwargs
+        Additional keyword arguments passed to the plotting function. Supported arguments depend on the backend:
+        - For `ms_matplotlib`: See matplotlib documentation.
+        - For `ms_bokeh`: See bokeh documentation.
+        - For `ms_plotly`: See plotly documentation.
+
+    Returns
+    -------
+    Figure
+        The plotted figure.
+
+    Raises
+    ------
+    ValueError
+        If `x` or `y` is not provided.
+
+    Examples
+    --------
+    >>> import pandas as pd
+    >>> import pyopenms_viz as oms_viz
+    >>> data = pd.DataFrame({'rt': [1, 2, 3], 'intensity': [10, 20, 30]})
+    >>> oms_viz.plot_chromatogram(data, x='rt', y='intensity', backend='ms_plotly')
+    """
+    if x is None or y is None:
+        raise ValueError("Both `x` and `y` must be provided.")
+    return data.plot(x=x, y=y, kind="chromatogram", backend=backend, **kwargs)
+
+
+def plot_spectrum(data: pd.DataFrame, x: str, y: str, backend: Optional[str] = None, **kwargs):
+    """
+    Plot a mass spectrum using a seaborn-like API.
+
+    Parameters
+    ----------
+    data : pandas.DataFrame
+        The data to plot.
+    x : str
+        The column name for the x-axis data.
+    y : str
+        The column name for the y-axis data.
+    backend : str, optional
+        The backend to use for plotting. If not provided, it will try to use
+        matplotlib, bokeh, or plotly in that order.
```

Comment:
As I said before, this is not currently implemented. Please implement or make backend argument mandatory.